### ISSUE TITLE
[Feature Request] Support for json5

### ISSUE THREAD

**Issue by Klaboe** [boothen/Json-Eclipse-Plugin#33]
A sub-set of json, [json5 ](https://json5.org) is a more relaxed version, aimed to be easier to use by humans.

It would be awesome if this plugin could provide support for this as well; and i think ANTLR4 supports the grammer.

**Comment by msteiger** [comment]
Very interesting - thanks for the hint. Do you know if there's a grammar available for ANTLR4 ?

**Comment by Klaboe** [comment]
I _think_ so 😅

There seems to be a json5 lexer parser here: http://lab.antlr.org but im not sure if that is all you need or not

Seems to be found here: https://github.com/antlr/grammars-v4/blob/master/json5/JSON5.g4

**Comment by Klaboe** [comment]
Did you have a chance to look into this?

**Comment by msteiger** [comment]
No, sorry. I use IntelliJ now most of the time and don't work on this plugin anymore. If you want to create a PR, I would review and test it, though.

**Comment by Klaboe** [comment]
Traitor 😆
